tools/wasme/cli/pkg/deploy/istio/istio_provider.go

```diff
@@ -160,7 +160,7 @@ func (p *Provider) ApplyFilter(filter *v1.FilterSpec) error {
 			return err
 		}
 		if err := abi.DefaultRegistry.ValidateIstioVersion(abiVersions, istioVersion); err != nil {
-			return errors.Errorf("image %v not supported by istio version %v", image.Ref(), istioVersion)
+			return errors.Errorf("image %v version %v not supported by istio version %v", image.Ref(), abiVersions, istioVersion)
```

Comment:
It looks like this change is breaking the istio tests which were expecting the original string:

![image](https://user-images.githubusercontent.com/1928125/112320661-9992b600-8c85-11eb-88ca-502c316ba2a9.png)